build/mitre_attack_checker/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "falco_mitre_attack_checker"
+version = "0.1.0"
+description = "Audit module to validate Falco rules against the Mitre ATT&CK Framework"
+authors = ["Louis Cailliot <louis.cailliot@thalesgroup.com>"]
```

Comment:
@leogr do we usually also add "The Falco Authors"? Or would it make sense here to add "The Falco Authors" as secondary author?

Reply:
You are right, this section is generated by poetry during the module creation but it is certainly wrong for this repo. I am waiting for your guidance.

Reply:
We use `The Falco Authors` only.

Reply:
And `cncf-falco-dev@lists.cncf.io` is the email.

Reply:
done ! thanks